easybuild/easyconfigs/f/FreeSurfer/FreeSurfer-6.0.0-centos6_x86_64.eb

```diff
@@ -0,0 +1,34 @@
+name = 'FreeSurfer'
+version = '6.0.0'
+versionsuffix = '-centos6_x86_64'
+
+homepage = 'http://freesurfer.net/'
+description = """FreeSurfer is a software package for the analysis and visualization of structural and functional 
+ neuroimaging data from cross-sectional or longitudinal studies. It is developed by the Laboratory for Computational 
+ Neuroimaging at the Athinoula A. Martinos Center for Biomedical Imaging."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%(namelower)s-Linux%(versionsuffix)s-stable-pub-v%(version)s.tar.gz']
+source_urls = ['ftp://surfer.nmr.mgh.harvard.edu/pub/dist/%(namelower)s/%(version)s']
+
+license_text = """email@example.com
+00000
+g1bb3r1sh"""
+
+modextravars = {
+    'FSFAST_HOME': '%(installdir)s/fsfast', 
+    'FUNCTIONALS_DIR': '%(installdir)s/sessions',
+    'FSF_OUTPUT_FORMAT': 'nii',
+    'MNI_DIR': '%(installdir)s/mni',
+    'MINC_BIN_DIR': '%(installdir)s/mni/bin',
+    'MINC_LIB_DIR': '%(installdir)s/mni/lib',
+    'MNI_DATAPATH': '%(installdir)s/mni/data',
+    'PERL5LIB': '%(installdir)s/mni/lib/perl5/5.8.5',
+    'FS_OVERRIDE': '0',
+    'FMRI_ANALYSIS_DIR': '%(installdir)s/fsfast',
```

Comment:
@rtripath89 lets sort these alphabetically?

Also, if you have a reference for what these mean, please include a comment above the `modextravars` line

Some of these should be handled via `modextrapaths`, no? `PERL5LIB` for example?